### ISSUE TITLE
🐛 Bug Report: Empty vars return false instead of empty dictionary

### ISSUE THREAD

**Issue by Pastajello** [appwrite/appwrite#3571]
### 👟 Reproduction steps

When a function has no set variables it is displayed correctly as 
![image](https://user-images.githubusercontent.com/28345098/180037473-7fda48d7-0f87-4bec-b61e-51f89c8fe664.png)
but what is returned by Functions(client).list is:
![image](https://user-images.githubusercontent.com/28345098/180037552-2886c17d-3568-4ad2-bca6-4dbaf698f145.png)
thus producing an exception in mapping:
![image](https://user-images.githubusercontent.com/28345098/180037655-854cc690-bd41-487f-854c-7d4c8801a372.png)

### 👍 Expected behavior

Empty vars should return empty dictionary, method works correctly

### 👎 Actual Behavior

empty vars return false, method throws an exception

### 🎲 Appwrite version

Version 0.15.x

### 💻 Operating system

Linux

### 👀 Have you spent some time to check if this issue has been raised before?

- [X] I checked and didn't find similar issue

### 🏢 Have you read the Code of Conduct?

- [X] I have read the [Code of Conduct](https://github.com/appwrite/appwrite/blob/HEAD/CODE_OF_CONDUCT.md)

**Comment by stnguyen90** [comment]
Thanks for reporting this! :pray: This is when using the Appwrite Flutter SDK, right?

**Comment by Pastajello** [comment]
@stnguyen90 yup, Flutter SDK 6.0.0. But I'm not sure if its SDK dependent since the json comes straight from web client. But who knows.
